rusoto/core/src/region.rs

```diff
@@ -28,6 +28,7 @@ pub enum Region {
     UsWest1,
     UsWest2,
     CnNorth1,
+    Local(&'static str)
```

Comment:
The .NET SDK calls this `Unknown`, but also uses it when converting the region from a string, which this does not. I can't find the similar functionality in the other SDKs to compare with. If you're familiar with this in another SDK, can you explain a bit how it works?